.github/pull_request_template.md

```diff
@@ -24,7 +24,7 @@ Issue Number: close #xxx
 
 Problem Summary:
 
-### What is changed and how it works?
+### What changed and how does it work?
```

Comment:
```suggestion
### What has changed and how does it work?
```

Reply:
<img width="802" alt="image" src="https://github.com/pingcap/tidb/assets/29879298/c92c3724-3eb8-44f8-8ba8-2d36c9ca6e04">
It seems we don't need it 😆